chia/wallet/trading/trade_store.py

```diff
@@ -78,6 +108,28 @@ async def create(
                 )
             )
 
+            await conn.execute(
+                (
+                    "CREATE TABLE IF NOT EXISTS coin_of_interest_to_trade_record("
```

Comment:
is it no possible to add an index to the database to fill the function of this new table?
It might be more space efficient to let sqlite take care of the indexing

Reply:
given one trade record can point to many coins it would require an `array` like data type for a proper index to be applied. Jsonb data type is an option, but was introduced this year which might affect our backwards compatibility.